kong/tools/utils.lua

```diff
@@ -1552,4 +1501,12 @@ do
 end
 
 
+do
+  local yield = require "kong.tools.yield"
+  for name, func in pairs(yield) do
+    _M[name] = func
+  end
+end
+
+
```

Comment:
As we're planning to split up `utils.lua` even further, make this function into a utility function that can be called like:

`include_module(_M, "kong.tools.yield")`

or similar to prevent code duplication.

Reply:
good idea, I will do it in another PR.